src/cp_control_utils.F

```diff
@@ -2277,6 +2279,19 @@ SUBROUTINE read_rtp_section(dft_control, rtp_section)
       dft_control%rtp_control%delta_pulse_direction = tmp
       CALL section_vals_val_get(rtp_section, "SC_CHECK_START", &
                                 i_val=dft_control%rtp_control%sc_check_start)
+      tmp_section => section_vals_get_subs_vals(rtp_section, "PRINT%PROJECTION_MO")
```

Comment:
Maybe use a more useful variable name, like `proj_mo_section`.

Reply:
Should be fixed